examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Aligned Sentence Pairs (Spanish -> German):\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "from fastdtw import fastdtw\n",
+    "\n",
+    "eng_spa_distance, eng_spa_alignment = fastdtw(eng_embeddings, spa_embeddings, dist=cosine)\n",
+    "eng_deu_distance, eng_deu_alignment = fastdtw(eng_embeddings, deu_embeddings, dist=cosine)\n",
+    "spa_deu_distance, spa_deu_alignment = fastdtw(spa_embeddings, deu_embeddings, dist=cosine)\n",
+    "\n",
+    "def print_aligned_sentences(alignment, sent1_list, sent2_list, lang1_name, lang2_name):\n",
+    "    print(f\"Aligned Sentence Pairs ({lang1_name} -> {lang2_name}):\")\n",
+    "    for idx1, idx2 in zip(alignment[0], alignment[1]):\n",
+    "        print(f\"{lang1_name}: {sent1_list[idx1]}\")\n",
+    "        print(f\"{lang2_name}: {sent2_list[idx2]}\")\n",
+    "        print(\"-\" * 40)\n",
+    "\n",
+    "print_aligned_sentences(eng_spa_alignment, eng_sentences, spa_sentences, \"English\", \"Spanish\")\n",
```

Comment:
I don't think we need all the three alignments here. 
I would print just one (as an example), but would include a larger number of sentences, and would maybe print their pairwise similarities, just for illustration (and to give some feeling of what makes their values larger or smaller).

Reply:
Also, just to increase the readability, you may put them into e.g. a pandas dataframe, so that the matched English and Spanish sentences are displayed side-by-side.